src/main/java/utils/CommonFuntions.java

```diff
@@ -0,0 +1,69 @@
+package utils;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class CommonFuntions {
+    //public static WebDriver driver;
+    public static WebDriver driver;
+
+
+    public enum driverType{
+        firefox, chrome
+    }
+
+    public static void goToUrl (WebDriver driver, String url, String property){
+        driver.get(url + property);
+
+    }
+
+    public static void createDriver (driverType type){
+        switch(type)
+        {
+            case chrome:
+                System.setProperty("webdriver.chrome.driver","src/test/resources/drivers/chromedriver");
+                driver = new ChromeDriver();
+
+            case firefox:
+                System.setProperty("webdriver.gecko.driver","src/test/resources/drivers/geckodriver");
+                DesiredCapabilities capabilities = DesiredCapabilities.firefox();
+                capabilities.setCapability("marionette", true);
+                driver = new FirefoxDriver(capabilities);
+
+        }
+    }
+
+
+    public static WebDriver getChromeDriver() {
+
+
+        System.setProperty("webdriver.chrome.driver","src/test/resources/drivers/chromedriver");
+
+        return new ChromeDriver();
+    }
+
+
+    public static void closeDriver() {
```

Comment:
Avoid having commented code. If it is not required either delete the method or dont use it.

Reply:
Done